docs-en/chapter_data_structure/classification_of_data_structure.md

```diff
@@ -2,9 +2,12 @@
 
 Common data structures include arrays, linked lists, stacks, queues, hash tables, trees, heaps, and graphs. They can be classified into two dimensions: "Logical Structure" and "Physical Structure".
 
+
```

Comment:
Redundant empty line

Reply:
thanks for the review Krahets, I will try fix it asap.